.github/workflows/build-preflight-in-a-box.yml

```diff
@@ -0,0 +1,107 @@
+name: New Tag
+on: push
+
+jobs:
+  build:
+    # This job only runs for issue comments
+    name: Update package
```

Comment:
This is the name of the job, usually I name it the same as the thing above ("Build and push Docker image")